programs/openbook-v2/src/state/orderbook/heap.rs

```diff
@@ -15,21 +15,21 @@ pub const NO_NODE: u16 = u16::MAX;
 /// starting at free_head while used nodes form a circular doubly-linked list starting at
 /// used_head.
 #[account(zero_copy)]
-pub struct EventQueue {
-    pub header: EventQueueHeader,
+pub struct EventHeap {
+    pub header: EventHeapHeader,
     pub nodes: [EventNode; MAX_NUM_EVENTS as usize],
     pub reserved: [u8; 64],
 }
 const_assert_eq!(
-    std::mem::size_of::<EventQueue>(),
+    std::mem::size_of::<EventHeap>(),
     16 + MAX_NUM_EVENTS as usize * (EVENT_SIZE + 8) + 64
 );
-const_assert_eq!(std::mem::size_of::<EventQueue>(), 91280);
-const_assert_eq!(std::mem::size_of::<EventQueue>() % 8, 0);
+const_assert_eq!(std::mem::size_of::<EventHeap>(), 91280);
```

Comment:
@binyebarwe how much SOL is this? please add a comment to the code

Reply:
https://explorer.solana.com/tx/4VdjqRDasTVT2u6eJ6mh3w8c3ykhf4SU7j7RPDYbLFsXTByN1ziDzupV3c7iSHno7ZJuEUj8HMpncjDwybxxGKXH?cluster=devnet